.travis.yml

```
@@ -1,79 +1,20 @@
-language: php
+version: ~> 1.0
 
-dist: xenial
-
-services:
-  - mysql
-  - postgresql
-  - xvfb
-
-addons:
-  apt:
-    packages:
-      - tidy
-      - chromium-chromedriver
-      - chromium-browser
-
-install:
-  - sudo apt-get remove -y --purge google-chrome-stable || true
+import:
+  - silverstripe/silverstripe-travis-shared:config/provision/standard-jobs-fixed-behat.yml
 
 env:
   global:
-    - COMPOSER_ROOT_VERSION=4.x-dev
-    - SS_BASE_URL="http://localhost:8080/"
-    - SS_ENVIRONMENT_TYPE="dev"
-    - RECIPE_CMS_VERSION=4.x-dev
-
-matrix:
-  fast_finish: true
-  include:
-    - php: 7.2
-      env: DB=PGSQL PHPUNIT_TEST=1 PHPCS_TEST=1
-    - php: 7.3
-      env: DB=MYSQL PHPUNIT_TEST=1
-    - php: 7.3
-      env: DB=MYSQL PDO=1 PHPUNIT_COVERAGE_TEST=1
-    - php: 7.4
-      env: DB=MYSQL PHPUNIT_TEST=1
-    - php: 7.3
-      env: DB=MYSQL ASSETADMIN_TEST=1
-  # TODO https://github.com/silverstripe/silverstripe-asset-admin/issues/1163
-  allow_failures:
-    - php: 7.3
-      env: DB=MYSQL BEHAT_TEST=1
+    - COMPOSER_ROOT_VERSION="4.x-dev"
+    - REQUIRE_RECIPE="4.x-dev"
+    - REQUIRE_EXTRA="silverstripe/frameworktest:0.1.0"
```

```suggestion
    - REQUIRE_EXTRA="silverstripe/frameworktest:0.3.0"
```
0.3.0 is the graphl 4 compatible version of frameworktest